### PR TITLE
v0.7.0

### PR DESCRIPTION
**Pull request by bhsiao55**
* Adjusting support to move cursor to the end of last text block.
* Fix formatting of `jwplayer_synced_at`.
* issue #232: add custom `blockRenderMap` to avoid creation of unsupported block types when pasting content into blocksmith.
* Update promotion-details-fields to include `slots` schema field.
* Use `@gdbots/schemas` v1.6.6
* Use `@triniti/schemas` 1.1.11

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # CHANGELOG
 
 
+## v0.6.4
+* Adjusting support to move cursor to the end of last text block.
```
Comment (efjacobson) [inline CHANGELOG.md:5]: @bhsiao55 i believe this change is associated with https://github.com/wb-apps/tmz-cms/issues/922

Comment (bhsiao55) [reply CHANGELOG.md]: @efjacobson I changed to match the title from https://github.com/wb-apps/tmz-cms/issues/922

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # CHANGELOG
 
 
+## v0.6.4
+* Cursor goes to top of content blocks or top of last block when using keyboard arrows to move to the end of the content blocks.
+* Fix formatting of `jwplayer_synced_at`.
+* issue #232: add custom `blockRenderMap` to avoid creation of unsupported block types when pasting content into blocksmith.
```
Comment (efjacobson) [inline CHANGELOG.md:7]: @bhsiao55 oh dang i forgot we were in triniti. can you remove this issue # from the scottish repo? thanks

Comment (bhsiao55) [reply CHANGELOG.md:7]: @efjacobson that issue is in triniti https://github.com/triniti/cms-js/issues/232

**Comment by gdbrown** [comment]
@bhsiao55 i'd like to also get @albertgcsula promotion slots pr in here too.  @albertgcsula is that ready?

**Comment by arunkarnati** [comment]
@bhsiao55 - can we also get the gdbots and triniti schema updates in this release, please.

**Comment by albertgcsula** [comment]
@gdbrown it's in code review. I can assign to @efjacobson for review.

**Comment by bhsiao55** [comment]
@gdbrown promotion slot pr has been included.

**Pull request merged**